deployment/azuredeploy.json

```diff
@@ -416,6 +426,25 @@
                 }
             }
         },
+        {
+            "type": "Microsoft.KeyVault/vaults/providers/diagnosticSettings",
+            "apiVersion": "2017-05-01-preview",
+            "name": "[concat(variables('keyVaultName'), '/Microsoft.Insights/Send to all locations')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.KeyVault/vaults', variables('keyVaultName'))]"

```

Comment:
Why are we not depending it on `AuditLogStorageAccountName` as well?

Reply:
seems like that got missed. thanks for pointing it out. adding it in the next commit.